packages/marketplace/src/components/ReadmePage.tsx

```diff
@@ -70,9 +70,12 @@ class ReadmePage extends React.Component<
     // Make the page scrollable
     const main = document.querySelector("#marketplace-readme")?.closest("main");
     if (main) {
-      setTimeout(() => {
+      const callScrollbar = setInterval(() => {
         // TODO: see if it's possible to use some load event or mutation observer to do this
+        main.style.overflowY = "visible";
         main.style.overflowY = "auto";
+        if (!document.querySelector(".os-scrollbar-vertical.os-scrollbar-unusable") || !main)
```

Comment:
What's the condition doing here? If there is a scrollbar or if there's no main, stop checking?

Reply:
When scrollbar is not present, the scrollbar element will have `os-scrollbar-unusable` in class list.
This checks if scrollbar is usable/visible or the user is no longer in the readme page (just to be sure).
Removing the `os-scrollbar-unusable` class name doesn't make it visible tho btw.

Reply:
Ah, alright 👍